Python/cim-convert-tool/configSshCim17.py

```diff
@@ -0,0 +1,191 @@
+                'cim:EnergyConsumer.p': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ActivePower.value'},
+                'cim:EnergyConsumer.q': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ReactivePower.value'}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:ControlArea': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:ControlArea.netInterchange': {'type': 'float', 'list': False}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:CurrentLimit': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:CurrentLimit.value': {'type': 'float', 'list': False}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:Disconnector': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:Equipment.inService': {'type': 'bool', 'list': False},
+                'cim:Switch.locked': {'type': 'bool', 'list': False},
+                'cim:Switch.open': {'type': 'bool', 'list': False}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:EquivalentInjection': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:Equipment.inService': {'type': 'bool', 'list': False},
+                'cim:EquivalentInjection.p': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ActivePower.value'},
+                'cim:EquivalentInjection.q': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ReactivePower.value'}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:Fuse': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:Equipment.inService': {'type': 'bool', 'list': False},
+                'cim:Switch.locked': {'type': 'bool', 'list': False},
+                'cim:Switch.open': {'type': 'bool', 'list': False}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:GeneratingUnit': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:Equipment.inService': {'type': 'bool', 'list': False}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:LinearShuntCompensator': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
```

Comment:
ShuntCompensator.sections and RegulatingCondEq.controlEnabled CIM attributes are missing (used in MV file)